docs/get-started/conclusion.md

```diff
@@ -1,3 +1,12 @@
 ---
 title: 'Conclusion'
 ---
+
+Congratulations! You learned the basics. Storybook is the most popular tool for UI component development and documentation. You’ll be able to transfer these skills to thousands of companies that use Storybook to build UIs including GitHub, Airbnb, and Stripe.
+
+If you’d like to learn workflows for building app UIs with Storybook, check out the in-depth guides on [Learn Storybook](https://www.learnstorybook.com/). Continue reading for detailed information on how to use Storybook APIs.
+
+What to learn next:
+
+- Write stories
+- Write component docs
```

Comment:
I guess these should be links

Reply:
i thought about it and forgot to link them. It's fixed now and the items are now links